### PR TITLE
Document JS Options

### PR DESCRIPTION
**Pull request by soulofmischief**
Had to dig into the module to find out how to pass options to the Watcher.

Hopefully this will save others some time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,9 +118,16 @@ watcher.run();
 Run your custom script:
 
 ```sh
-node watch-it.js src/input.scss dist/output.css "ie >= 9, > 1%" 
+node watch-it.js src/input.scss dist/output.css "ie >= 9, > 1%"
 ```
 
+Available options are a subset of the CLI options:
+
+* includePaths
```
Comment (kottenator) [inline README.md]: Please, wrap with backticks.

**Review by kottenator — CHANGES_REQUESTED** [review]

**Comment by soulofmischief** [comment]
Funny, I thought if I went with backticks you would tell me to make it a list.

Is this more what you wanted, or do you want me to copy over the descriptions as well?

**Comment by kottenator** [comment]
Not exactly what I wanted :)

Sorry, I wasn't clear enough with my request: could you, please, make it _as a list_ but _each keyword in backticks_?

Like:

```md
* `one`
* `two`
* `three`
```

**Comment by soulofmischief** [comment]
Haha, of course. The dark deed you've requested is done, sir.

**Review by kottenator — APPROVED** [review]

**Pull request merged**